nixos/modules/services/networking/radicale.nix

```diff
@@ -84,9 +161,43 @@ in
         ));
         User = "radicale";
         Group = "radicale";
+        StateDirectory = "radicale/collections";
+        # Hardening
+        CapabilityBoundingSet = "";
+        DeviceAllow = [ "/dev/stdin" ];
+        DevicePolicy = "strict";
+        IPAddressAllow = mkIf bindLocalhost "localhost";
+        IPAddressDeny = mkIf bindLocalhost "any";
```

Comment:
Is this problematic? One scenario I can think of is a hook accessing the (non-local) network. Does anyone do that?

Reply:
I wouldn't feel comfortable putting a hostname in there, but maybe that's just the networker in me speaking. But making it contingent on `bindLocalhost` sounds like a sensible approach.

Reply:
It's not actually a hostname, it's a special name interpreted as `127.0.0.0/8 ::1/128` by systemd: https://www.freedesktop.org/software/systemd/man/systemd.resource-control.html#IPAddressAllow=ADDRESS%5B/PREFIXLENGTH%5D%E2%80%A6